src/modules/EventModule.cpp

```diff
@@ -462,7 +462,7 @@ EventModule::EventModule(UmikoBot* client) : Module("event", true)
 		auto& config = getServerEventData(channel.guildId());
 		if (configRD.m_TicketIds.size() == 0)
 		{
-			client.createMessage(message.channelId(), "**You have no tickets.**\nYou can buy them by using `!buyticket <amount>`");
+			client.createMessage(message.channelId(), "**You have no tickets.**\nYou can buy them by using `!buytickets <amount>`");
```

Comment:
Same as earlier. `!` may not be the chosen prefix!

Reply:
Okay done 🙂